src/frontend/planner_test/tests/testdata/output/over_window_function.yaml

```diff
@@ -580,16 +617,21 @@
         └─LogicalOverWindow { window_functions: [row_number() OVER(PARTITION BY t.y ORDER BY t.x ASC ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)] }
           └─LogicalProject { exprs: [t.x, t.y, t._row_id] }
             └─LogicalScan { table: t, columns: [t.x, t.y, t._row_id] }
+  batch_plan: |-
+    BatchExchange { order: [], dist: Single }
+    └─BatchProject { exprs: [t.x, t.y] }
+      └─BatchFilter { predicate: (3:Int32 < row_number) AND (row_number = 6:Int32) AND (row_number <= 5:Int32) }
+        └─BatchOverWindow { window_functions: [row_number() OVER(PARTITION BY t.y ORDER BY t.x ASC ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)] }
+          └─BatchExchange { order: [t.y ASC, t.x ASC], dist: HashShard(t.y) }
+            └─BatchSort { order: [t.y ASC, t.x ASC] }
```

Comment:
The Exchange can not make the `OverWindow`'s computing can not be shuffled well because the partition key in a period could be the same.
a better plan might be 
```
        └─BatchOverWindow { window_functions: [row_number() OVER(PARTITION BY t.y ORDER BY t.x ASC ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)] }
            └─BatchSort { order: [t.y ASC, t.x ASC] }
                     └─BatchExchange { dist: HashShard(t.y) }

```